packages/astro/src/@types/astro.ts

```diff
@@ -2275,6 +2275,10 @@ export interface AstroUserConfig {
 		 * export const collections = { blog, dogs };
 		 * ```
 		 *
+	   * :::note
+		 * Loaders will not automatically [exclude files prefaced with an `_`](/en/guides/routing/#excluding-pages). Use a regular expression such as `pattern: '**/[^_]*.md'` in your loader to to ignore these files.
+		 * :::
+		 *																																																																																																																		 
```

Comment:
```suggestion
		 *	
```
I'm not sure if this line full of hidden tabs was causing a problem, or where it came from, but I guess can't hurt to remove. :sweat_smile: 

![image](https://github.com/user-attachments/assets/2368e9ef-b736-43e6-8182-f69488f6b505)